### ISSUE TITLE
Input is shown and placeholder is not inline with span with same placeholder text

### ISSUE THREAD

**Issue by chaseadamsio** [joeblau/gitignore.io#132]
In Chrome this is obviously broken:
<img width="1552" alt="screen shot 2015-08-26 at 6 25 28 am" src="https://cloud.githubusercontent.com/assets/1024544/9494553/5e9bc12e-4bbb-11e5-81f6-4d0e0e509aee.png">

In Firefox only obvious if you know to look for it (there's a faint shadow over the textbox:
<img width="1552" alt="screen shot 2015-08-26 at 6 26 09 am" src="https://cloud.githubusercontent.com/assets/1024544/9494555/623dc91c-4bbb-11e5-87c8-505c50cd7a5b.png">

In Safari not broken:
<img width="1376" alt="screen shot 2015-08-26 at 6 26 31 am" src="https://cloud.githubusercontent.com/assets/1024544/9494557/6834d18a-4bbb-11e5-9dd1-3f40c1ac1321.png">

**Comment by joeblau** [comment]
Yeah I saw that last week when I checked on Chrome :).  I'll take a look at that tonight, hopefully it's just a simple issue with the Select2 box that is fixed in an update.  Thanks for pointing this out.

**Comment by chaseadamsio** [comment]
:+1: if you don't get a chance to get to it, I'm happy to look into it.

**Comment by joeblau** [comment]
Sure.  If you submit a PR I'll merge it.

**Comment by andrewtpoe** [comment]
I'm attempting to fork / fix this issue, but running into another when attempting to serve this locally. Here is my process so far...
Forked project
Cloned locally
npm start

The error I am receiving is: 
```
Server listening on http://localhost:8000
Application ready to serve requests.
Environment: development
Thu, 27 Aug 2015 15:21:37 GMT uncaughtException ENOENT, no such file or directory '/Users/techapoe/gitignore.io/models/../data/custom/AppCode.gitignore'
Error: ENOENT, no such file or directory '/Users/techapoe/gitignore.io/models/../data/custom/AppCode.gitignore'
    at Error (native)
    at Object.fs.openSync (fs.js:500:18)
    at Object.fs.readFileSync (fs.js:352:15)
    at /Users/techapoe/gitignore.io/models/datastore.js:32:31
    at FSReqWrap.oncomplete (fs.js:95:15)
```
After reporting this error the server shuts down. Looking through my local files /data/custom/AppCode.gitignore is not present, even though it is present in my forked repository.

Is this a known issue or can you advise on how to remedy? I'm fairly certain I have a fix for the double placholder issue above, but I would like to test it more thoroughly.

**Comment by joeblau** [comment]
When you clone the repository, make sure you do `git clone --recursive git@github.com:joeblau/gitignore.io.git`.  This repository uses [GitHub's master list](https://github.com/github/gitignore) of templates as a git submodule so AppCode.gitignore is a symlink to Objective-C or Xcode in GitHub's repo.

**Comment by andrewtpoe** [comment]
This is how the home page now appears in Chrome on initial load with my pull request's updates:

<img width="723" alt="screen shot 2015-08-27 at 3 07 50 pm" src="https://cloud.githubusercontent.com/assets/11183459/9530397/84b09584-4ccd-11e5-9bef-ea0b49bcabdf.png">

And in Firefox:

<img width="743" alt="screen shot 2015-08-27 at 3 09 23 pm" src="https://cloud.githubusercontent.com/assets/11183459/9530416/a6d37780-4ccd-11e5-9da4-205051fcb317.png">

**Comment by joeblau** [comment]
Awesome :+1: - Will merge PR when I get back to my computer.

**Comment by mcandre** [comment]
I'm seeing this bug again. It's noticeable in Firefox, and much more so in Chrome.

http://i.imgur.com/tSJKKK1.png

**Comment by chaseadamsio** [comment]
Yeah, I'm wondering if there was a regression?

**Comment by cassidoo** [comment]
I'm seeing this still, too, on Chrome 45.0.2454.99:
![image](https://cloud.githubusercontent.com/assets/1454517/10559920/9eac91ea-74c8-11e5-860a-5f42c89ea5d6.png)

**Comment by joeblau** [comment]
This was fixed by PR #123.
